mscore/palettebox.cpp

```diff
@@ -552,6 +553,15 @@ bool PaletteBox::eventFilter(QObject* obj, QEvent *event)
                   if (keyEvent->key() == Qt::Key_Up || keyEvent->key() == Qt::Key_Down ||
                           keyEvent->key() == Qt::Key_Enter || keyEvent->key() == Qt::Key_Return) {
                         navigation(keyEvent);
+                        for (Palette* p : palettes()) {
+                              if (p->getCurrentIdx() != -1) {
+                                    QString acc_descr = p->cellAt(p->getCurrentIdx())->name;
+                                    _searchBox->setAccessibleDescription(acc_descr);
```

Comment:
Also, are you sure that the _searchbox accessible description will be reset when the user will use it again?

Reply:
No, I don't think it will be reset.

Reply:
that's a problem then? When the searchbox will loose the focus, and the user will keep on working and then go back the the searchbox, it will not have the right information. Correct?

Reply:
Yes, it won't have the right information. I will fix this by resetting the accessible description in the focusOutEvent function.